frontend/src/js/tooltip/Tooltip.tsx

```diff
@@ -164,9 +175,21 @@ const Tooltip = () => {
         />
         <Head>
           <PinnedLabel>
-            <TypeIcon icon={isFolder ? faFolder : faMinus} />
+            <TypeIcon
+              icon={
+                isFolder || parent !== label
+                  ? hasChild
```

Comment:
A little hard to understand: `hasChild` is also checking `parent !== label`,  right? Could you make this a little easier to read / understand?